vaadin-popover-flow-parent/vaadin-popover-flow/src/main/java/com/vaadin/flow/component/popover/Popover.java

```diff
@@ -528,7 +528,7 @@ public void setFocusDelay(int focusDelay) {
 
     /**
      * The delay in milliseconds before the popover is opened on target focus.
-     * Defaults to {@code 0}.
+     * Defaults to {@code 500}.
```

Comment:
This is still somewhat confusing, because `getFocusDelay` returns `0` by default. [In this PR](https://github.com/vaadin/flow-components/pull/6828) we actually return the default value of the web component as well, though I guess that is not consistent everywhere.

Reply:
Changed to make it clear that `500` is used when the delay is not specified (this is how the web component works). I'm not sure if we should change the `getFocusDelay()` etc as they also return `0` in Tooltip.